# Pylint import-error bug with setuptools >=64.0.0

https://github.com/PyCQA/pylint/issues/7306

On debian bullseye, python 3.9.2

To reproduce:

```bash
./run.sh
```
